src/bin/pgcopydb/string_utils.c

```diff
@@ -35,6 +35,21 @@ intToString(int64_t number)
 	return intString;
 }
 
+/*
+ * uint32ToString converts a 32-bit int to an UInt32String, which contains a
+ * decimal string representation of the integer.
+ */
+UInt32String
+uint32ToString(uint32_t number)
+{
+	UInt32String intString;
+
+	intString.intValue = number;
+
+	sformat(intString.strValue, UINT32STRING_MAX_DIGITS, "%" PRId32, number);
+
+	return intString;
+}
```

Comment:
Can we just use the existing `intToString` function here? It takes as an argument a `int64_t` number, and an OID (`uint_32t`) should fit within that range right?